pkg/labels/labels_test.go

```diff
@@ -453,3 +453,35 @@ func TestLabels_Compare(t *testing.T) {
 		testutil.Equals(t, test.expected, got, "unexpected comparison result for test case %d", i)
 	}
 }
+
+func TestLabels_Has(t *testing.T) {
+	labels := Labels{
```

Comment:
can you rename `labels` so as not to confuse with the package? just l should be fine

might also help to move it below the tests struct defintion right before the for loop